src/style.css

```diff
@@ -1,3 +1,103 @@
-body {
+body,
+h1,
+ul,
+li,
+p,
+a {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Don't use '*' selector (it impacts performance). Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser. Consider applying the `box-sizing: border-box;` to all elements using a more specific selector like `html`.